Seed.Api.Tests/Controllers/UserControllerTest.cs

```diff
@@ -114,28 +117,6 @@ public class UserControllerTest
             userService.VerifyAll();
         }
 
-        [Fact]
-        public async void Create_ShouldReturnNotFound_WhenNoUserWasNotCreated()
```

Comment:
Do you mind adding a test for the bad request scenario?

Reply:
there's one, for the null case, the other cases are data anotations, and should be covered with the validateModel attribute

Reply:
Ok